UPGRADE-5.0.md

```diff
@@ -681,4 +681,4 @@ WebProfilerBundle
 WebServerBundle
 ---------------
 
- * The bundle has been removed.
+ * The bundle has been deprecated and can be installed separately. You may also use the `Symfony Local Web Server` instead.
```

Comment:
why using backticks?

Reply:
indeed useless, a habit on personal doc to highlight some wording :)